src/js/content_scripts/input-mapper.js

```diff
@@ -153,7 +153,7 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
                     },
                     // Right thumbstick vertical axis.
```

Comment:
Given the way the controls appear on screen (back arrow, forward arrow), I'd argue that you should use the right thumbstick horizontal axis for this by default.